www/assets/css/theme.css

```diff
@@ -92,6 +92,10 @@ body {
 }
 
 main {
+  display: block; /* using 'flex' here would mess with `margin: auto` blocks */
```

Comment:
I’d appreciate some help here.

Setting `display: flex` on `<main>` would be neat to balance the main content in short pages (e.g. 404, Duck Typist…) but it messes with block elements carrying a `margin: [whatever] auto` property (e.g. `<figure>` elements) and I don’t understand why.

Does anyone have a clue here ?

Reply:
Not a direct answer to this particular problem, but it looks to me that all that is to compensate the fact that the main `nav` is not inside the `header` tag, and so their flex container (`body`) will separate them if you don’t force `main` to take all the space.
So, just put the nav inside the header, no?

Reply:
oh, that’s because of the sticky…

Reply:
can’t you put the sticky on the whole header, but not make it start at 0, so that only the nav will stick ?

Reply:
Nope, that won’t work either. Tried it already. :-)

Reply:
otherwise you can add a "width: 100%" on your figures...